libs/blocks/tabs/tabs.js

```diff
@@ -84,6 +87,77 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
+// Check if current tab is not visible and next tab is visible
+function previousTab(item, i, arr) {
+  const next = arr[i + 1];
+  return (!isTabInTabListView(item) && next && isTabInTabListView(next));
```

Comment:
nit: `next` should be first in check so if false we don't run `isTabInTabListView`

Reply:
Updated.